### PR TITLE
Removed prismjs files from repo and made it as an external dependency

### PR DESCRIPTION
**Pull request by ibhi**
Removed prismjs files from this repository and moved it as an dependency in package.json. In this way no need to update this repositories prismjs files whenever prismjs releases new version or adds new features/plugins.
Note: Also this repository code and the code installed from npm are not the same. The npm version doesnt have the code for line numbers support.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 const fs = require('hexo-fs');
 const path = require('path');
-const Prism = require('./prism.js');
+const Prism = require('prismjs');
+require('prismjs/plugins/line-numbers/prism-line-numbers.min');
```
Comment (ele828) [inline index.js]: Is this line neccessary? It seems like it's not used in the context.

File: index.js [hunk 2]
```diff
@@ -50,17 +53,17 @@ function PrismPlugin(data) {
   fs.copyFile(themeUri, path.join(baseDir, 'public', 'css', themeFile));
 
   if (mode === 'realtime') {
-    fs.copyFile(path.join(pluginDir, 'prism.js'),
+    fs.copyFile(path.join(prismDir, 'prism.js'),
       path.join(baseDir, 'public', 'js', 'prism.js'));
     if (line_number) {
-      fs.copyFile(path.join(pluginDir, 'plugin', 'line-numbers.js'),
-        path.join(baseDir, 'public', 'js', 'line-numbers.js'));
+      fs.copyFile(path.join(prismDir, 'plugins/line-numbers', 'prism-line-numbers.js'),
```
Comment (ele828) [inline index.js]: I guess we could use `.min.js` file

File: index.js (hunk 1 above)
Comment (ele828) [inline index.js]: It requires a book of Prism? Just out of curious, thanks!

File: index.js [hunk 3]
```diff
@@ -91,12 +94,12 @@ function PrismPlugin(data) {
   if (mode === 'realtime') {
     jsImports = `<script src="${configRoot}/js/prism.js"></script>`;
     if (line_number) {
-      jsImports += `<script src="${configRoot}/js/line-numbers.js"></script>`;
+      jsImports += `<script src="${configRoot}/js/prism-line-numbers.js"></script>`;
```
Comment (ele828) [inline index.js]: I guess we could use .min.js file

File: index.js (hunk 2 above)
Comment (ibhi) [inline index.js]: Yes I agree. I will update it to use min.js

File: index.js (hunk 3 above)
Comment (ibhi) [inline index.js]: Yes, I agree, I will update it to use min.js

File: index.js (hunk 1 above)
Comment (ibhi) [inline index.js]: line-numbers require statement is necessary to load the plugin in to context. Without that line, line numbers are not working.

Comment (ele828) [inline index.js]: gotcha, thanks

**Comment by ele828** [comment]
After testing this changes, I found out it doesn't work in some specific languages like `jsx`, `objective-c` , etc. 

I investigated a little bit and personally think it's because your didn't import all of those predefined language supports of Prism.js

So for now, I temporarily reverts the changes, and if you have any solutions, feel free to commit it. I will also look at it these days.

Thanks!

**Pull request merged**